docs/ontology_search/ontology_search_introduction.md

```diff
@@ -3,23 +3,40 @@
 The ontology search functionality can be accessed from within a project via the application drawer.
 Therefore, start by [navigating](#ontology-navigation) to the ontology summary view.
 
-The ontology search allows you to [search](#ontology-search) for ontology information 
-within our database. 
+The ontology search allows you to [search](#ontology-search) for ontology information
+within our database.
 
 ## Ontology Navigation
 
-From the landing page you can navigate into a project via the project list by clicking on the project card directly.
+From the landing page you can navigate into a project via the project list by clicking on the
+project card directly.
 
-Once within the project open the application drawer via the button on the top left of the application
+Once within the project open the application drawer via the button on the top left of the
+application
 and select "ontology search" to navigate to the ontology summary view.
 ![ontology_summary.png](images/ontology_search_summary.png)
 
 ## Ontology Search
 
-Start by providing at least 2 letters in the ontology search field of the entity for which you want to retrieve the ontology information. 
-Once done so a card list showing information for all ontology entries which contain the provided letters will appear.
+Start by providing at least 2 letters in the ontology search field of the entity for which you want
+to retrieve the ontology information.
+Once done so a card list showing information for all ontology entries which contain the provided
+letters will appear.
 ![ontology_search_triggered.png](images/ontology_search_triggered.png)
 
-Additionally, it allows you to copy the ontology [CURIE](https://link.springer.com/article/10.1007/s12599-022-00744-0),
-which can be especially handy during [measurement registration](../measurement/measurement_registration.md)
+!!! note "Enable species search"
+    The application provides access to species terminologies for the
+    complete [tree of life](https://www.ncbi.nlm.nih.gov/taxonomy) provided by NCBI.
+    If activated, only the species taxonomy is queried. 
+
+??? note "Why not including species in on search?" 
+    We currently host an own database instance for efficient queries of species to support
+    faster lookups. Also, the TIB terminology service currently does not provide
+    the tree of life, which we have integrated for all other term lookups.
```

Comment:
Bezieht sich das "which" hier auf den tree of life oder auf TIB? Das ist etwas unklar imho.